src/olympia/addons/tasks.py

```diff
@@ -290,11 +290,7 @@ def update_addon_hotness(averages):
     log.info('[%s] Updating add-ons hotness scores.', (len(averages)))
 
     averages = dict(averages)
-    addons = (
-        Addon.objects.filter(guid__in=averages.keys())
-        .filter(status__in=amo.APPROVED_STATUSES)
-        .no_transforms()
-    )
+    addons = Addon.unfiltered.filter(guid__in=averages.keys()).no_transforms()
```

Comment:
This also removed the filter for `status__in=amo.APPROVED_STATUSES`. Is that intentional?

Reply:
Yes. We want to set `hotness` for all kinds of add-ons, regardless of status